apps/studio/prisma/schema.prisma

```diff
@@ -221,3 +222,33 @@ model Whitelist {
 
   @@index([email])
 }
+
+enum AuditLogEvent {
+  ResourceCreate
+  ResourceUpdate
+  ResourceDelete
+  ResourceMove
```

Comment:
question: do we need this `ResourceMove`? Seems to be covered under `ResourceUpdate` with the before and after parentId being recorded in the `metadata` field.

Reply:
hmm, actually you're right - because we record the info fully, i think we can omit this! good catch!